db/migrate/20171107172507_fix_whitehall_edition_first_published_at.rb

```diff
@@ -0,0 +1,30 @@
+require "csv"
+
+class FixWhitehallEditionFirstPublishedAt < ActiveRecord::Migration[5.1]
+  def up
+    count_first_published = 0
+    start_datetime = "2016-02-29 09:24:09"
+    end_datetime = "2016-02-29 09:24:11"
+
+    data = CSV.read(
+      Rails.root.join(
+        "db", "migrate", "data", "whitehall_editions_first_published_at.csv"
+      )
+    )
+
+    data.each do |content_id, _, first_published_at|
```

Comment:
I'm not sure if this is completely worth it, but since we're doing this similar block of code in each migration and since it's possible we may need it in the future, I wonder if it's worth extracting a lot of this out into a [helper](https://github.com/alphagov/publishing-api/tree/master/db/migrate/helpers)?

Reply:
Yeah I wondered that too. We have some slight annoyances like this having a document_id as second argument. I'll have a quick look over it.